src/vs/platform/issue/electron-main/issueMainService.ts

```diff
@@ -442,6 +442,26 @@ export class IssueMainService implements IIssueMainService {
 		});
 	}
 
+	async $sendReporterStatus(extensionId: string, extensionName: string): Promise<boolean[]> {
```

Comment:
nit: the other functions are called "get" and their message is called "trigger" and their reply is called "trigger...response". This one is kinda flipping that... and it's inconsistent.

In both cases, you're triggering something in the window process and returning it to the issue reporter process.